Tools/Scripts/webkitpy/w3c/test_downloader.py

```diff
@@ -115,7 +115,10 @@ def checkout_test_repository(self, revision, url, directory):
             git = self.git(directory)
         _log.info('Checking out revision ' + revision)
         git.checkout(revision, not self._options.verbose)
-        self.upstream_revision = git.rev_parse('HEAD')
+        self.upstream_revision = self.get_rev_parse_head(git)
+
+    def get_rev_parse_head(self, git):
```

Comment:
I don't understand the purpose of this function, instead of just using `git.rev_parse('HEAD')`, since that's all this function does.

Reply:
@JonWBedard it's to be able to use `git.rev_parse` in `test_importer.py` without having to import the full git just getting the hash. So I'm using the one in `test_downloader.py`

Reply:
That said I could probably call it directly too since I have initialized git. Good point.

Reply:
ok fixed. Thanks.